server/test/unit/server/webservices/views/test_utils.py

```diff
@@ -0,0 +1,46 @@
+import json
+import unittest
+
+from django.http import HttpResponse, HttpResponseNotFound
+
+from pulp.server.webservices.views import utils
+
+
+class TestWebservicesUtils(unittest.TestCase):
+    """
+    Test webservices utilities.
+    """
+
+    def test_generate_django_response_default_params(self):
+        """
+        Make sure that the response is correct under normal conditions.
+        """
+        test_content = """{'foo': 'bar'}"""
```

Comment:
I think this might not be doing quite what you intended. Your sample content looks suspiciously like serialized json. That will work, but it's really just the same as being any old string. I think what you might actually want is a regular python object here, such as

``test_content = {'foo': 'bar'}``

which will get serialized into a json string during your test.